### PR TITLE
Add VUID 01655

### PR DESCRIPTION
**Pull request by sfricke-samsung**
VUID-VkSamplerYcbcrConversionCreateInfo-ycbcrModel-01655

### CODE REVIEW

File: layers/parameter_validation_utils.cpp [hunk 1]
```diff
@@ -5174,7 +5175,55 @@ bool StatelessValidation::ValidateCreateSamplerYcbcrConversion(VkDevice device,
...
+
+        if ((channels < 4) && ((components.r == VK_COMPONENT_SWIZZLE_A) || (components.g == VK_COMPONENT_SWIZZLE_A) ||
+                               (components.b == VK_COMPONENT_SWIZZLE_A))) {
+            skip |= LogError(device, vuid,
+                             "%s: The ycbcrModel is not VK_SAMPLER_YCBCR_MODEL_CONVERSION_RGB_IDENTITY so components.r (%s), "
+                             "components.g (%s), nor components.b (%s) can't be VK_COMPONENT_SWIZZLE_A.",
+                             apiName, string_VkComponentSwizzle(components.r), string_VkComponentSwizzle(components.g),
+                             string_VkComponentSwizzle(components.b));
+        } else if ((channels < 3) &&
+                   ((components.r == VK_COMPONENT_SWIZZLE_B) || (components.g == VK_COMPONENT_SWIZZLE_B) ||
```
Comment (Tony-LunarG) [inline layers/parameter_validation_utils.cpp:5213]: If components.[r,g,b] != VK_COMPONENT_SWIZZLE_B but components.b == VK_COMPONENT_SWIZZLE_IDENTITY the log message is confusing

Comment (sfricke-samsung) [reply layers/parameter_validation_utils.cpp:5213]: well when I wrote "equivalent to VK_COMPONENT_SWIZZLE_B" it was intended to have case where `b` was `VK_COMPONENT_SWIZZLE_IDENTITY`

I tried coming up with a better way to word the log message, did you have any suggestions?

Comment (sfricke-samsung) [reply layers/parameter_validation_utils.cpp:5213]: what about

```
"%s: The ycbcrModel is not VK_SAMPLER_YCBCR_MODEL_CONVERSION_RGB_IDENTITY so components.r (%s), "
"components.g (%s), nor components.b (%s) can't be VK_COMPONENT_SWIZZLE_B 
 (components.b also can't be VK_COMPONENT_SWIZZLE_IDENTITY).",

```

Comment (Tony-LunarG) [reply layers/parameter_validation_utils.cpp:5213]: That is much clearer to me.

**Pull request merged**